commit ccb9ab7ba94d7b005f4fd820f1cb91caf73dbf89
Author: HiassofT <github@hias.horus.com>
Date:   Wed Apr 6 21:45:01 2016 +0200

    Revert "bcm2835-dma: Fix dreq not set for slave transfers"
    
    This reverts commit 8ad957e866a1fe1450f663f2b00a57d7de44904c.
    
    
    
    DMA channels are set in devicetree, thus dreq will be set,
    
    and this commit is no longer needed.
    
    
    
    Signed-off-by: Matthias Reichl <hias@horus.com>

diff --git a/drivers/dma/bcm2835-dma.c b/drivers/dma/bcm2835-dma.c
index 985019b..d26b6bd 100644
--- a/drivers/dma/bcm2835-dma.c
+++ b/drivers/dma/bcm2835-dma.c
@@ -679,8 +679,6 @@ static int bcm2835_dma_slave_config(struct dma_chan *chan,
 	}
 
 	c->cfg = *cfg;
-	if (!c->dreq)
-		c->dreq = cfg->slave_id;
 
 	return 0;
 }
diff -uN a/1.txt b/1.txt
--- a/dummy/rpi_1231_ccb9ab7ba94d7b005f4fd820f1cb91caf73dbf89.txt	1970-01-01 00:00:00.000000000 +0000
+++ b/dummy/rpi_1231_ccb9ab7ba94d7b005f4fd820f1cb91caf73dbf89.txt	2013-12-23 04:07:40.000000000 +0000
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
